server/main.py

```diff
@@ -45,26 +48,29 @@ class Payload(BaseModel):
         "radiance",
     ]
     database: Literal["hitran", "geisa", "hitemp"]
+    wavelength_units: Literal["1/u.cm", "u.nm"]
+    pressure_units: Literal["u.bar", "u.mbar", "u.atm", "u.torr", "u.mTorr", "u.Pa"]
+    pathLength_units: Literal["u.cm", "u.m", "u.km"]
```

Comment:
```suggestion
    path_length_units: Literal["u.cm", "u.m", "u.km"]
```
to be consistent with snake case